optuna/visualization/_pareto_front.py

```diff
@@ -288,3 +295,42 @@ def _make_hovertext(trial: FrozenTrial) -> str:
         indent=2,
     )
     return text.replace("\n", "<br>")
+
+
+def _make_marker(
+    trials: List[FrozenTrial],
+    include_dominated_trials: bool,
+    use_constraints_func: bool,
+    dominated_samples: bool = False,
```

Comment:
In the Optuna codebase, we usually use the term `trial` instead of `sample`. How about rephrasing it?
In addition, `is_best` seems easier for me to understand the role of this argument (although the polarity is opposite). What do you think of it?

```suggestion
    dominated_trials: bool = False,
```